lnrpc/devrpc/dev_server.go

```diff
@@ -342,3 +346,25 @@ func (s *Server) ImportGraph(ctx context.Context,
 
 	return &ImportGraphResponse{}, nil
 }
+
+// Quiesce initiates the quiescence process for the channel with the given
+// channel ID. This method will block until the channel is fully quiesced.
+func (s *Server) Quiesce(_ context.Context, in *QuiescenceRequest) (
```

Comment:
squash with previous commit adding to protos (won't compile without the impl)

Reply:
So I tried running the `check-each-commit` script and am not having any build issues. How did you repro?

Reply:
I didn't actually test it, just assumed that the previous commit would fail on our [compile time check](https://github.com/lightningnetwork/lnd/blob/master/lnrpc/devrpc/dev_server.go#L70) that `Server` implements `DevServer`, because the `Quiesce` method is missing. 

But you're right it does compile, which kinda baffled me. Turns out that's covered by `UnimplementedDevServer` being embedded means that the check doesn't really do anything anymore.